lib/puppet/pops/types/string_converter.rb

```diff
@@ -385,6 +386,21 @@ def initialize
   # | s         | same as d
   # | p         | same as d
   #
+  # ### Binary (value)
+  #
+  # | Format    | Default formats
+  # | ------    | ---------------
+  # | s         | binary as unquoted characters
+  # | p         | 'Binary("<base64strict>")'
+  # | b         | '<base64>' - base64 string with newlines inserted
+  # | B         | '<base64strict>' - base64 strict string (without newlines inserted)
```

Comment:
We could use a different letter if that helps. The intent is that when you use %s you want the binary as text. Normally binary data would not be text and you would spew out binary gobbledygook. So the distinction needs to exist. The `to_s` variant is typically used blindly in all sorts of "lets print it out" so, it can not be the "binary as text" option.